clients/config-kie-client/src/main/java/org/apache/servicecomb/config/kie/client/model/KieConfiguration.java

```diff
@@ -22,6 +22,9 @@ public class KieConfiguration {
 
   private int pollingWaitInSeconds;
 
+  private int refreshInterval;
```

Comment:
give a default value too. Other frameworks may missed the setting when integrate this version.